game-app/game-core/src/main/java/games/strategy/triplea/attachments/PoliticalActionAttachment.java

```diff
@@ -152,22 +150,62 @@ public List<RelationshipChange> getRelationshipChanges() {
         .collect(Collectors.toList());
   }
 
-  private RelationshipChange parseRelationshipChange(final String encodedRelationshipChange) {
+  private RelationshipChange parseRelationshipChange(final String encodedRelationshipChange)
+      throws IllegalStateException {
```

Comment:
```suggestion
     {
```
IllegalStateException is an unchecked exception. There is no effect in declaring a method throwing an unchecked exception (although it can be done if you __really__ want to emphasize and document that the method throws. Still, that is probably better done with javadoc)